packages/fhir-converter/src/lib/handlebars-converter/handlebars-helpers.js

```diff
@@ -899,8 +921,8 @@ module.exports.external = [
   {
     name: "isNaN",
     description: "Checks if the object is not a number using JavaScript isNaN: isNaN object",
-    func: function (o) {
-      return isNaN(o);
+    func: function(str) {
+      return isNaN(str);
     },
```

Comment:
isn't this supposed to be an object? otherwise does the comment need to be updated?

Reply:
yah should not have updated that one my bad